src/app/account/invitation/[inviteId]/new-user-account-form.tsx

```diff
@@ -53,25 +71,30 @@ const SetupAccountForm: FC<InviteProps & { onComplete(): void }> = ({ inviteId,
     })
 
     const { mutate: createAccount, isPending: isCreating } = useMutation({
-        mutationFn: (form: FormValues) => onCreateAccountAction({ inviteId, form }),
+        mutationFn: (formData: FormValues) => onCreateAccountAction({ inviteId, email, form: formData }), // Pass email
         onError: handleMutationErrorsWithForm(form),
-        async onSuccess(_, vals) {
+        async onSuccess(clerkUserId, vals) {
+            // clerkUserId is returned by onCreateAccountAction
             if (!signIn) {
-                reportError('unable to signin')
+                reportError(new Error('SignIn object not available'), 'SignIn not available post account creation')
```

Comment:
`reportError` will accept and display a string:
```suggestion
                reportError('SignIn not available post account creation')
```